apps/discord-bot/utils/scheduledEventFunctions.ts

```diff
@@ -25,15 +25,11 @@ export const createLiveStreamEventFromMessage = (
   const url = getURL(params.platformDomain, msgContent);
 
   const firstMsgEmbedTitle = message.embeds[0]?.title;
-  const urlAndMentionRegex = /@(everyone|here)|\bhttps?:\/\/\S+/g;
-  const processedMsgContent = msgContent.replace(urlAndMentionRegex, "").trim();
 
-  let eventName = "";
+  let eventName = msgContent.substring(0, 20);
```

Comment:
not really sure what this line does? can you actually move this to a helper function and name it accordingly, so it's easier to understand and straight forward to update what it does later.

Reply:
this line is to prevent the eventName variable from being an empty string in case the if condition in the bottom line is not met, because the event title cannot be an empty string.